tests/integration/conftest.py

```diff
@@ -0,0 +1,137 @@
+# Copyright 2021 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Fixtures for LXD integration tests."""
+import os
+import pathlib
+import random
+import string
+import subprocess
+import sys
+import tempfile
+
+import pytest
+
+from craft_providers import lxd, multipass
+
+
+def generate_instance_name():
+    return "itest-" + "".join(random.choices(string.ascii_uppercase, k=8))
+
+
+@pytest.fixture()
+def home_tmp_path():
+    """Multipass doesn't have access to /tmp."""
```

Comment:
this is more of a code comment than a docstring for the test, right?

Reply:
Fixed!